e2e_samples/dataset_versioning/infra/README.md

```diff
@@ -69,6 +71,11 @@ terraform plan -var "name=myapp" -out=/tmp/myapp
 terraform apply /tmp/myapp
 ```
 
+### Other options
```

Comment:
nit: maybe add CICD on the section title to be more clear?
also, we have this line:
  environment: ${{ parameters.buildingEnvironment }}
If people dont set the env, will it throw error? if so, let's mention setup enviroment here as well :)

Reply:
Let me update the whole readme as different issue for cleaning up.